app.js

```diff
@@ -4,9 +4,18 @@ const PORT      = process.env.PORT || 8080;
 
 const path      = require('path'); 
 const express   = require('express');
+const bodyParser = require('body-parser');
 const exphbs    = require('express-handlebars');
 const fs        = require('fs');
 const app       = express();
+const saveUser  = require('./modules/datastore.js');
+
+// parse application/x-www-form-urlencoded
+app.use(bodyParser.urlencoded({ extended: false }));
+
+// parse application/json
+app.use(bodyParser.json());
```

Comment:
since we ended up just using express.json, i think you can remove references to bodyParser